src/main/java/uk/gov/hmcts/reform/sscs/model/single/hearing/PartiesNotifiedResponse.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.reform.sscs.model.single.hearing;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.joda.time.DateTime;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class PartiesNotifiedResponse {
```

Comment:
Minor: can we change the ordering of these to match the swagger spec (usual json convention is alphabetical order as well like the spec has)